x-pack/plugins/cloud_security_posture/public/components/cloud_security_data_table/cloud_security_data_table.tsx

```diff
@@ -0,0 +1,275 @@
+/*
```

Comment:
yes, this can be configured by the user in the settings, however in the next [PR](https://github.com/elastic/kibana/pull/167844/files) we changed the default to be single line with text truncation